.github/workflows/dev-image-update.yaml

```diff
@@ -69,21 +70,23 @@ jobs:
           DISABLE_ESLINT_PLUGIN: true
         run: |
           rm -rf jade-dev-account.pem
-          docker build -t gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:${GCR_TAG} --build-arg CACHEBUST=$(date +%s) .
+          docker build -t gcr.io/${gcr_google_project}/jade-data-repo-ui:${GCR_TAG} --build-arg CACHEBUST=$(date +%s) .
       # Push the Docker image to Google Container Registry
       - name: Publish and tag new docker container to GCR
         run: |
           gcloud auth activate-service-account --key-file jade-dev-account.json
           gcloud auth configure-docker --quiet
-          docker push gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:${GCR_TAG}
+          docker push gcr.io/${gcr_google_project}/jade-data-repo-ui:${GCR_TAG}
           gcloud container images \
             add-tag \
-            gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${GCR_TAG}" \
-            gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${{ steps.uiprevioustag.outputs.tag }}" --quiet
+            gcr.io/${gcr_google_project}/jade-data-repo-ui:"${GCR_TAG}" \
+            gcr.io/${gcr_google_project}/jade-data-repo-ui:"${PREV_TAG}" --quiet
           gcloud container images \
             add-tag \
-            gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${GCR_TAG}" \
-            gcr.io/${{ env.gcr_google_project }}/jade-data-repo-ui:"${GCR_TAG}-develop" --quiet
+            gcr.io/${gcr_google_project}/jade-data-repo-ui:"${GCR_TAG}" \
+            gcr.io/${gcr_google_project}/jade-data-repo-ui:"${GCR_TAG}-develop" --quiet
+        env:
+          PREV_TAG: ${{ steps.uiprevioustag.outputs.tag }}
```

Comment:
Were you able to test these changes? I'm a little worried that it won't pick up on these variables with the changes - I've found it to be really picky in the past. 
I think I've been able to test a subset of this action by commenting out the bumperstep, helm_tag_bump an cherry_pick_image_to_production_gcr tasks and then kicking off the run on your branch.

Reply:
Yup, I confirmed it works!